GameStore.Services/Services/Implementation/GenreServices.cs

```diff
@@ -47,6 +47,19 @@ public GenreModel GetItemById(Guid id)
             }
         }
 
+        public GenreInfoTransferModel GetGenreInfoTransferById(Guid id)
+        {
+            var genre = genreRepository.GetItemById(id);
+            if (genre == null)
+            {
```

Comment:
Let's try to implement this as ternary operator

Reply:
Changed condition operator